crates/red_knot_workspace/src/site_packages.rs

```diff
@@ -100,55 +236,335 @@ fn site_packages_dir_from_sys_prefix(
 
         path.push("site-packages");
         if system.is_directory(&path) {
-            tracing::debug!("Resolved site-packages directory to '{path}'");
             return Ok(path);
         }
     }
     Err(SitePackagesDiscoveryError::NoSitePackagesDirFound)
 }
 
+/// The various ways in which parsing a `pyvenv.cfg` file could fail
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum PyvenvCfgParseErrorReason {
```

Comment:
```suggestion
pub enum PyvenvCfgParseErrorKind {
```

That's the terminology that e.g. `std::io::Error` uses